lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -11,12 +11,40 @@
         'instructor.views.api.register_and_enroll_students', name="register_and_enroll_students"),
     url(r'^list_course_role_members$',
         'instructor.views.api.list_course_role_members', name="list_course_role_members"),
+    url(r'^list_course_sections$',
+        'instructor.views.api.list_course_sections', name="list_course_sections"),
+    url(r'^list_course_problems$',
+        'instructor.views.api.list_course_problems', name="list_course_problems"),
+    url(r'^list_course_tree$',
+        'instructor.views.api.list_course_tree', name="list_course_tree"),
     url(r'^modify_access$',
         'instructor.views.api.modify_access', name="modify_access"),
     url(r'^bulk_beta_modify_access$',
         'instructor.views.api.bulk_beta_modify_access', name="bulk_beta_modify_access"),
     url(r'^get_grading_config$',
         'instructor.views.api.get_grading_config', name="get_grading_config"),
+    url(r'^get_all_students(?P<make_csv>/csv)?$',
+        'instructor.views.api.get_all_students', name="get_all_students"),
+    url(r'^save_query$',
+        'instructor.views.api.save_query', name="save_query"),
+    url(r'^get_saved_queries$',
+        'instructor.views.api.get_saved_queries', name="get_saved_queries"),
+    url(r'^get_temp_queries$',
+        'instructor.views.api.get_temp_queries', name="get_temp_queries"),
+    url(r'^get_single_query/(?P<inclusion>\S{2,3})/(?P<query_type>(Section|Problem))/(?P<state_type>\S+)/(?P<state_id>\S+)$',
+        'instructor.views.api.get_single_query', name="get_single_query"),
+    url(r'^get_single_query',
```

Comment:
write a comment here about this being purely a convenience, b/c we don't know the parameters yet at the time of 'reverse'